cli-feed-v3.json

```diff
@@ -2063,6 +2063,53 @@
           "sha2": "c74dd3e17144b04139b668b9ba528ca19bca4f21a2233d4b555ebe962e4b6da5"
         }
       ]
+    },
+    "2.31.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.27",
+      "cli": "https://functionscdn.azureedge.net/public/2.7.1524/Azure.Functions.Cli.min.win-x86.2.7.1524.zip",

```

Comment:
Looks like something went wrong. This is the link to the last CLI release `2.30.0`. The template did not update properly from Pipelines.

Reply:
I will do a re-release and update PR if it fixes it

Reply:
A re-build fixed it